pom.xml

```diff
@@ -101,7 +101,7 @@
     <version.com.google.guava.for-selenium>23.0</version.com.google.guava.for-selenium>
 
     <version.com.google.elemental2>1.0.0-beta-1</version.com.google.elemental2>
-    <version.org.jboss.errai>4.6.0-SNAPSHOT</version.org.jboss.errai>
+    <version.org.jboss.errai>4.7.0-SNAPSHOT</version.org.jboss.errai>
```

Comment:
what about starting to use fixed version here
not the 4.7.0-SNAPSHOT but released 4.6.0.Final

Reply:
fixed